src/stores/searchStore.js

```diff
@@ -12,6 +14,15 @@ export const useSearchStore = defineStore('searchStore', {
 	actions: {
 		setSearchText(text) {
 			this.searchText = text;
+
+			const query = { ...router.currentRoute.value.query, search: text };
```

Comment:
Is it duplicated code of tagsStore ?) 
What should we do in case.. "need to add one more sharing case (rating)". Should we copy-paste this code ? I think  it is not good :(